skimage/feature/tests/test_orb.py

```diff
@@ -69,6 +69,8 @@ def test_keypoints_orb_less_than_desired_no_of_keypoints():
     assert_almost_equal(exp_cols, detector_extractor.keypoints[:, 1])
 
 
+@xfail(condition=arch32,
+       reason='https://github.com/scikit-image/scikit-image/issues/3091')
```

Comment:
Here I'd like a more verbose message: "Known test failure on 32-bit platforms. See [link] for details."